site/content/docs/5.1/components/placeholders.md

```diff
@@ -4,6 +4,7 @@ title: Placeholders
 description: Use loading placeholders for your components or pages to indicate something may still be loading.
 group: components
 toc: true
+added: "5.1"
```

Comment:
Are these used?

Reply:
This frontmatter tells the version picker in the subnav that if you're browsing v5.1.x and you want to see the placeholders page in the previous version from that dropdown (v5.0.2), you cannot, because the page wasn't added until v5.1.